### ISSUE TITLE
The 'sanic_restplus.api' package was not installed in a way that PackageLoader understands

### ISSUE THREAD

**Issue by hsz1273327** [ashleysommer/sanic-restplus#23]
win10,sanic 20.12.3,sanic-restplus 0.5.6

I got the value error as below:

```code
Traceback (most recent call last):
  File "app/__main__.py", line 12, in <module>
    main(sys.argv[1:])
  File "app/__main__.py", line 8, in main
    app(argv)
  File "C:\Users\hsz12\Documents\WORKSPACE\project_manager_with_git\tp_py_sanic_api\env\lib\site-packages\schema_entry\entrypoint.py", line 204, in __call__
    self.parse_args(parser, argv)
  File "C:\Users\hsz12\Documents\WORKSPACE\project_manager_with_git\tp_py_sanic_api\env\lib\site-packages\schema_entry\entrypoint.py", line 451, in parse_args
    self.do_main()
  File "C:\Users\hsz12\Documents\WORKSPACE\project_manager_with_git\tp_py_sanic_api\app\app.py", line 92, in do_main
    rest_assoc.api(api)
  File "C:\Users\hsz12\Documents\WORKSPACE\project_manager_with_git\tp_py_sanic_api\env\lib\site-packages\sanic_restplus\restplus.py", line 10, in api
    return plug.api(reg, *args, api_class=api_class, **kwargs)
  File "C:\Users\hsz12\Documents\WORKSPACE\project_manager_with_git\tp_py_sanic_api\env\lib\site-packages\sanic_restplus\restplus.py", line 76, in api
    api.init_api(reg, **kwargs)
  File "C:\Users\hsz12\Documents\WORKSPACE\project_manager_with_git\tp_py_sanic_api\env\lib\site-packages\sanic_restplus\api.py", line 225, in init_api
    self._init_app(app, context)
  File "C:\Users\hsz12\Documents\WORKSPACE\project_manager_with_git\tp_py_sanic_api\env\lib\site-packages\sanic_restplus\api.py", line 235, in _init_app
    render_api_fn = self._setup_jinja2_renderer()
  File "C:\Users\hsz12\Documents\WORKSPACE\project_manager_with_git\tp_py_sanic_api\env\lib\site-packages\sanic_restplus\api.py", line 282, in _setup_jinja2_renderer
    loader = PackageLoader(__name__, 'templates')
  File "C:\Users\hsz12\Documents\WORKSPACE\project_manager_with_git\tp_py_sanic_api\env\lib\site-packages\jinja2\loaders.py", line 309, in __init__
    raise ValueError(
ValueError: The 'sanic_restplus.api' package was not installed in a way that PackageLoader understands.
```

**Comment by luk1999** [comment]
As a temporary fix you can force `Jinja2==2.10.3` in you requirements file.

**Comment by hsz1273327** [comment]
thanks
